istio-control/istio-config/templates/deployment.yaml

```diff
@@ -36,9 +36,7 @@ spec:
           - containerPort: 15019
           command:
           - /usr/local/bin/galley
-{{- if and (.Values.global.istioVersion) (semverCompare "<1.2" .Values.global.istioVersion) }}
```

Comment:
Yes, this was cherry-picked from 1.1 into master https://github.com/istio/istio/commit/f7feb0fcbab73d05375815f06956ac3002f231d6

Reply:
So for supporting 1.0 the switch would still be necessary?

@costinm should the installer strive for 1.0 support?

Reply:
I don't think we need to at this point. It did because it was started around 0.8. 

We do need to test that the installer can work alongside 1.0, i.e. if someone has 1.0 installed in istio-system they can deploy 1.1 or 1.2 using the new installer.